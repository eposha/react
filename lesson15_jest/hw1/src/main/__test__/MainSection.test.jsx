import React from "react";
import { shallow } from "enzyme";
import MainSection from "../MainSection";

describe("<MainSection/ >", () => {
  it("should display MainSection", () => {
    const wrappedComponent = shallow(<MainSection />);
    expect(wrappedComponent).toMatchSnapshot();
  });
});
