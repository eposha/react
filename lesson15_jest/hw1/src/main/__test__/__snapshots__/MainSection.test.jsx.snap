// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HolidayLine/ > should display HolidayLine 1`] = `
<div
  className="holiday-wrapper"
>
  <div
    className="holiday"
  >
    <div
      className="holiday__day"
      key="0"
    />
    <div
      className="holiday__day"
      key="1"
    />
    <div
      className="holiday__day"
      key="2"
    />
    <div
      className="holiday__day"
      key="3"
    />
    <div
      className="holiday__day"
      key="4"
    />
    <div
      className="holiday__day"
      key="5"
    />
    <div
      className="holiday__day"
      key="6"
    />
  </div>
</div>
`;
